Dockerfile

```diff
@@ -0,0 +1,53 @@
+# These may be overwritten by --build-arg to build promtail or debug images
+ARG APP=loki
+ARG TYPE=production
+
+# ca-certificates
+FROM alpine:3.9 as ssl
+RUN apk add --update --no-cache ca-certificates
+
+# use grafana/loki-build-image to compile binaries
+FROM grafana/loki-build-image as golang
+ARG GOARCH="amd64"
+COPY . /go/src/github.com/grafana/loki
+WORKDIR /go/src/github.com/grafana/loki
+RUN touch loki-build-image/.uptodate &&\
+    mkdir /build
+
+# production image
+FROM golang as builder-production
```

Comment:
should we ping a version ?

Reply:
No, because it does not actually refer to the `docker.io/library/golang` image here but rather to `docker.io/grafana/loki-build-image`:
https://github.com/grafana/loki/blob/f7987c0bd0bbe1f7b2e8475a410c8c6316c09391/Dockerfile#L10

Reply:
got it.